lms/static/js/utils/navigation.js

```diff
@@ -0,0 +1,73 @@
+var edx = edx || {},
+
+    Navigation = (function() {
+
+        var navigation = {
+
+            init: function() {
+
+                if ($('#accordion').length) {
+
+                    navigation.openAccordion();
+                }
+            },
+
+            openAccordion: function() {
+                $('#open_close_accordion a').click();
```

Comment:
This was pre-existing code for the course navigation. If this isn't triggered, the course nav doesn't "open". I'm hesitant to change or remove it as I'm not sure what else in the LMS uses it. Thoughts?